tools/testing-gui/src/pages/THORNamePage.tsx

```
@@ -0,0 +1,380 @@
+  const handleLookup = async () => {
+    if (!thorName) return
+
+    await lookupOp.execute(
+      async () => {
+        const { ThorchainQuery } = await import('@xchainjs/xchain-thorchain-query')
+        const thorchainQuery = new ThorchainQuery()
+
+        // Fetch THORName details and current block height in parallel
+        const [details, lastBlock] = await Promise.all([
+          thorchainQuery.getThornameDetails(thorName),
+          fetch('https://thornode.ninerealms.com/thorchain/lastblock').then(r => r.json()),
```

ESLint

[error] Replace `r` with `(r)`

(prettier/prettier)